package.json

```diff
@@ -77,7 +77,9 @@
     "fastboot-transform": "^0.1.3",
     "hammerjs": "^2.0.8",
     "propagating-hammerjs": "^1.4.6",
-    "resolve": "^1.3.3",
+    "fs-extra": "^4.0.1",
```

Comment:
@betocantu93 `fs-extra` and `rsvp` are likely not necessary after the PR refactor.

Reply:
yep